readme.md

```diff
@@ -336,3 +336,150 @@ In this exercise we've practised the basics queries in MongoShell also we've lea
 [Reference API MongoDB Driver](http://mongodb.github.io/node-mongodb-native/2.2/api/)
 [MongoDB Driver Website](https://mongodb.github.io/node-mongodb-native/)
 [Getting Started](http://mongodb.github.io/node-mongodb-native/2.2/quick-start/)
+
+
+//PART 1
```

Comment:
Que gusto da ver que alguien se molesta en leer y hacer las cosas como dios manda. Enhorabuena.

Reply:
Me refiero a que me mandáis la IT1 para corroborar el trabajo.